lib/src/utf16.dart

```diff
@@ -38,7 +44,13 @@ extension Utf16Pointer on Pointer<Utf16> {
   ///
   /// If [length] is provided, zero-termination is ignored and the result can
   /// contain NUL characters.
-  String toDartString({int? length}) {
+  ///
+  /// Returns `null` if this is a [nullptr].
+  String? toDartString({int? length}) {
```

Comment:
Changing the return type from `String` to `String?` is a breaking change.

This would warrant bumping package:ffi to `2.0.0`.

Reply:
Yes. That is a true statement. I already made it in the initial PR comment.